contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_checker.py

```diff
@@ -113,3 +110,30 @@ def test_syntax_error(self):
     target = self.make_target('a/python:error', PythonLibrary, sources=['error.py'])
     self.set_options(fail=False)
     self.assertEqual(1, self.execute_task(target_roots=[target]))
+
+  def test_lint_for_py3_only(self):
+    self.create_file('a/python/fail.py', contents=dedent("""
+                         x=2+3
+                         print(x+7)
+                       """))
+    target = self.make_target('a/python:fail', PythonLibrary, sources=['fail.py'])
+    with self.assertRaises(TaskError) as task_error:
+      self.execute_task(target_roots=[target])
+    self.assertIn('3 Python Style issues found', str(task_error.exception))
```

Comment:
String searching - assertIn here - or assertEqual (above) but not both.

Reply:
This is following prior art from the failure tests above. Do you want me to refactor all these?

Reply:
Urg - you're right. No, that's fine.